docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -325,7 +354,7 @@ Same as `CALL`, but allowing to interact with any contract. This will not check
         <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000001000</code>
     </p>
 
-Same as `STATICCALL`, but allowing to interact with any contract. This will not check for [**Allowed Calls**](#allowed-calls) if the caller has any of these restrictions set.
+Same as `STATICCALL`, but allowing to interact with any contract. This will not check for [**Allowed Calls**](#allowed-calls) if the controller has any of these restrictions set.
```

Comment:
```suggestion
Same as `STATICCALL`, but it allows interacting with any contract. This will not check if the controller has any   [**Allowed Calls**](#allowed-calls) restrictions set.
```